build/aclocal/gtk-4.0.m4

```diff
@@ -0,0 +1,208 @@
+# Configure paths for GTK+
```

Comment:
Where does `gtk-4.0.m4` come from?

Reply:
I made a copy of gtk-3.0.m4 and changed some things.